delfin/drivers/dell_emc/vmax/alert_handler.py

```diff
@@ -17,129 +17,130 @@
-                such as source system id, location, alarm id together can be
-                used to construct this
-    me_dn : Unique id at resource module (management system) side. me stands
-            for management element here
-    me_name : Unique name at resource module (management system) side
-    native_me_dn : Unique id of the device at source system that reports the
-                   alarm
-    location : Alarm location information. It helps to locate the lowest unit
-               where fault is originated(Name-value pairs)
-               ex: Location = subtrack, No = 1, Slot No = 5.
-                   shelf Id = 1, board type = ADSL
-    event_type : Basic classification of the alarm. Probable values such as
-                 status, configuration, topology ....
-    alarm_id : Identification of alarm
-    alarm_name : Name of the alarm, might need translation from alarm id
-    severity : Severity of alarm. Helps admin to decide on action to be taken
-               Probable values: Critical, Major, Minor, Warning, Info
-    device_alert_sn : Sequence number of alert generated. This will be helpful
-                      during alert clearing process
-    manufacturer : Vendor of the device
-    Product_name : Name of the product
-    probable_cause : Probable reason for alert generation
-    clear_type : Alarm clearance type such as manual, automatic, reset clear
-    me_category : Resource category of the device generating the alarm
-                  Probable value: Network,Server,Storage..
+    Alert Model	Description
+    Part of the fields filled from delfin resource info and other from driver
+    *****Filled from delfin resource info***********************
+    storage_id	Id of the storage system on behalf of which alert is generated
+    storage_name	Name of the storage system on behalf of which alert is
+                    generated
+    manufacturer	Vendor of the device
+    product_name	Product or the model name
+    serial_number	Serial number of the alert generating source
+    ****************************************************
+    *****Filled from driver side ***********************
+    alert_id	Unique identification for a given alert type
+    alert_name	Unique name for a given alert type
+    severity	Severity of the alert
+    category	Category of alert generated
+    type	Type of the alert generated
+    sequence_number	Sequence number for the alert, uniquely identifies a
+                              given alert instance used for clearing the alert
+    occur_time	Time at which alert is generated from device
+    detailed_info	Possible cause description or other details about the alert
+    recovery_advice	Some suggestion for handling the given alert
+    resource_type	Resource type of device/source generating alert
+    location	Detailed info about the tracing the alerting device such as
+                slot, rack, component, parts etc
+    *****************************************************
```

Comment:
move comments to driver.py and only describe the essential properties

Reply:
Updated